pilot/pkg/security/authz/model/model.go

```diff
@@ -265,3 +272,42 @@ func (m *Model) Generate(service *ServiceMetadata, forTCPFilter bool) *envoy_rba
 	}
 	return policy
 }
+
+// getPrincipalsIncludingAliases returns a list of users encoded in SPIFFE format with the local
+// trust domain and its aliases.
+// For example, for a user "bar" in namespace "foo".
+// If the local trust domain is "td2" and its alias is "td1" (migrating from td1 to td2),
+// getPrincipalsIncludingAliases returns ["td2/ns/foo/sa/bar", "td2/ns/foo/sa/bar]].
```

Comment:
Change the first "td2/ns/foo/sa/bar" to "td1/ns/foo/sa/bar"?

Reply:
Thanks. Since it's a typo, I'll wait for other reviews from @yangminzhu and @diemtvu before making the change.